fix: glossary remove term without a definition (#491)
o1-labs/docs2: docs/glossary.mdx

---
title: Glossary

> description: A zero knowledge list of terms with a focus on the Mina ecosystem. 

Reviewer: I think 'zero knowledge list' is fun but not so accurate and perhaps a bit of a stretch. What do you think of this suggestion?

```suggestion
description: A list of terms relating to zero knowledge and the Mina ecosystem. 
```

Author: way better, it's always better together, thank you!
